client/src/components/categorical/index.tsx

```diff
@@ -213,4 +193,12 @@ class Categories extends React.Component<{}, State> {
   }
 }
 
-export default Categories;
+function mapStateToProps(state: RootState): StateProps {
```

Comment:
super nit - I started putting these in the types file to make the component even smaller

Reply:
Oh that's right! Let me extract `types.ts` file too thank you!